README.md

```diff
@@ -3,11 +3,31 @@
 TODO write something meaningful here
 
 ## Development
+#### Prerequisites
+- Setup the [rust toolchain](https://www.rust-lang.org/tools/install)
+```rustup default stable```
+
+- Install leptos
+```cargo install cargo-leptos```
+
+- Install and setup postgres ([example for arch](https://wiki.archlinux.org/title/PostgreSQL))
+
+- With postgress running, create a database  `hive-local` and a user `hive-dev` as the owner
+```
+sudo -u postgres createuser hive-dev
+sudo -u postgres createdb -O hive-dev hive-local
+```
+
+## 
```

Comment:
After postgress is setup either do the nix stuff or:
Install diesel with
cargo install diesel_cli --no-default-features --features postgres
run the migrations
cd db
diesel migration run
cd ..
Serve the site with:
cargo leptos watch or cargo leptos watch --hot-reload

The nix part is nice but optional.